client/components/TestRenderer/OutputTextArea/index.jsx

```diff
@@ -0,0 +1,96 @@
+import React, { useEffect } from 'react';
+import PropTypes from 'prop-types';
+import styled from '@emotion/styled';
+import { Feedback } from '..';
+import { Form } from 'react-bootstrap';
+import { noOutputTextAreaValue } from './constants';
+
+const OutputTextAreaWrapper = styled.div`
+    > textarea {
+        width: 100%;
+    }
+`;
+
+const NoOutputCheckbox = styled(Form.Check)`
+    display: inline-block;
+    float: right;
+    color: ${props => (props.disabled ? 'lightgray' : 'inherit')};
```

Comment:
I felt like the lightgray might be TOO lightgray, I'm curious if you try 4:1 contrast (which is #7F7F7F) if it will look good and also sufficiently disabled? If not I think I'm okay dipping below 4:1 contrast, since the control is pretty irrelevant when there's text in the text box.